eureka2-ext/eureka2-eureka1x-rest-api/build.gradle

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
Can we standardise our naming here? we refer to 2.0 as eureka2 (no "x"), but then refer to eureka1 as eureka1x. May be more clear to change eureka1x to eureka1.